### PR TITLE
libcdio 2.1.0 Survey and kodi-pvr-iptvsimple update to 19.0.0

### PR DESCRIPTION
**Pull request by eatradish**
Topic Description
-----------------

libcdio update to 2.1.0 and update/rebuild for libcdio

Package(s) Affected
-------------------
groups/libcdio-rebuilds
groups/xine-lib-rebuilds
kodi-pvr-iptvsimple: 19.0.0
whipper: 10.0.0-1

Security Update?
----------------

No

Build Order
-----------

libcdio groups/libcdio-rebuilds groups/xine-lib-rebuilds kodi-pvr-iptvsimple whipper

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`   
- [x] AArch64 `arm64`
    

**Secondary Architectures**

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`

Update(s) Uploaded to Stable
----------------------------

**Primary Architectures**

- [ ] AMD64 `amd64`   
- [ ] AArch64 `arm64`

**Secondary Architectures**

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [ ] Loongson 3 `loongson3`
- [ ] PowerPC 64-bit (Little Endian) `ppc64el`

### CODE REVIEW

File: extra-multimedia/kodi/autobuild/beyond [hunk 1]
```diff
@@ -10,7 +10,7 @@ KODI_DEV_HEADERS="""
   kodi-visualization-dev
 """
 
-pushd ${SRCDIR}/build
+pushd ${SRCDIR}/abbuild
```
Comment (KexyBiscuit) [inline extra-multimedia/kodi/autobuild/beyond]: Use parentheses instead of `pushd`/`popd`/`cd`.

**Review by KexyBiscuit — CHANGES_REQUESTED** [review]
One change requested.

File: extra-multimedia/kodi/autobuild/beyond [hunk 2]
```diff
@@ -10,9 +10,9 @@ KODI_DEV_HEADERS="""
   kodi-visualization-dev
 """
 
-pushd ${SRCDIR}/build
+cd "$SRCDIR"/abbuild
```
Comment (KexyBiscuit) [inline extra-multimedia/kodi/autobuild/beyond:13]: Use parentheses instead of `pushd`/`popd`/`cd`.

**Review by KexyBiscuit — CHANGES_REQUESTED** [review]
One change requested.

**Review by cthbleachbit — APPROVED** [review]

**Review by KexyBiscuit — APPROVED** [review]
LGTM.

**Pull request merged**